src/Cassandra/Requests/StartupOptionsFactory.cs

```diff
@@ -0,0 +1,70 @@
+﻿// 
+//       Copyright (C) 2019 DataStax Inc.
+// 
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+// 
+//       http://www.apache.org/licenses/LICENSE-2.0
+// 
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+// 
+
+using System.Collections.Generic;
+using Cassandra.Helpers;
+
+namespace Cassandra.Requests
+{
+    internal class StartupOptionsFactory : IStartupOptionsFactory
+    {
+        public const string CqlVersionOption = "CQL_VERSION";
+        public const string CompressionOption = "COMPRESSION";
+        public const string NoCompactOption = "NO_COMPACT";
+        public const string DriverNameOption = "DRIVER_NAME";
+        public const string DriverVersionOption = "DRIVER_VERSION";
+
+        public const string DriverName = "DataStax C# Driver";
```

Comment:
Would it be possible to obtain this information from the assembly product name (`Title` in the `csproj` file), that way it won't require any adaptation on the DSE driver.

Reply:
good suggestion, changed it